test/functional/tests/example/example_test.py

```diff
@@ -147,3 +151,71 @@ def test_example_multidut():
                 TestRun.LOGGER.info(dut1_ex.run_expect_success("which casctl").stdout)
                 for name, disk in TestRun.disks.items():
                     TestRun.LOGGER.info(f"{name}: {disk.path}")
+
+
+@pytest.mark.require_disk("drbd_device", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.multidut(2)
+def test_drbd_example():
+    """
+    title: Example test using DRBD API.
+    description: Create primary and secondary resources on two DUTs using drbd.
+    pass_criteria:
+      - primary drbd resource created.
+      - secondary drbd resource created.
+    """
+    with TestRun.step("Prepare"):
+        dut1, dut2 = TestRun.duts
+        nodes = []
+        with TestRun.use_dut(dut1):
+            TestRun.dut.hostname = TestRun.executor.run_expect_success("uname -n").stdout
+            TestRun.LOGGER.info(TestRun.dut.hostname)
+
+            drbd_dev = TestRun.disks["drbd_device"]
+            drbd_md_dev = RamDisk.create(Size(100, Unit.MebiByte), 1)[0]
+            drbd_dev.create_partitions([Size(200, Unit.MebiByte)])
+            drbd_dev = drbd_dev.partitions[0]
+
+            nodes.append(
+                Node(TestRun.dut.hostname, drbd_dev.path, drbd_md_dev.path, dut1.ip, "7790")
+            )
+        with TestRun.use_dut(dut2):
+            TestRun.dut.hostname = TestRun.executor.run_expect_success("uname -n").stdout
+            TestRun.LOGGER.info(TestRun.dut.hostname)
+
+            drbd_dev = TestRun.disks["drbd_device"]
+            drbd_md_dev = RamDisk.create(Size(100, Unit.MebiByte), 1)[0]
+            drbd_dev.create_partitions([Size(200, Unit.MebiByte)])
+            drbd_dev = drbd_dev.partitions[0]
+
+            nodes.append(
+                Node(TestRun.dut.hostname, drbd_dev.path, drbd_md_dev.path, dut2.ip, "7790")
+            )
+        drbd_resurce_name = "caches"
+        caches = Resource(drbd_resurce_name, "/dev/drbd0", nodes)
```

Comment:
If we use drbd_resurce_name helper variable maybe adding drbd_exported_object as variable will be plus.

Reply:
One way or another 😅